jyellick/plob/chaincode/assetregistry.go

```diff
@@ -177,11 +180,11 @@ func (ac *assetContext) create() ([]byte, error) {
 }
 
 func (ac *assetContext) lock() ([]byte, error) {
-	if ac.functionArg == nil {
-		return nil, fmt.Errorf("Must pass toChannel argument")
+	if !(ac.functionArgs != nil && (len(ac.functionArgs) == 2)) {
+		return nil, fmt.Errorf("Must pass toChaincode/toChannel arguments")
 	}
 
-	toChannel := string(ac.functionArg)
+	toChaincode, toChannel := string(ac.functionArgs[0]), string(ac.functionArgs[1])
```

Comment:
Do you think it's valuable for the purpose of PoC to allow the chaincode to be specified? Seems easiest if we require that it's this chaincode in both channels?